python/python/ert_gui/plottery/plot_config.py

```diff
@@ -283,6 +280,6 @@ def copyConfigFrom(self, other):
 
     @classmethod
     def createCopy(cls, other):
-        copy = PlotConfig(None)
+        copy = PlotConfig(PlotSettings( ))
```

Comment:
If you do `self._plot_settings = plot_settings` in `__init__` 
then you can do: `copy = PlotConfig(other._plot_settings)` here